src/api/app/models/package.rb

```diff
@@ -891,6 +891,28 @@ def buildresults(prj = project)
     local_build_results
   end
 
+  def jobhistorylist(project, repository, arch, limit = 100)
+    results = Jobhistory.find_hashed(project: project.name, package: name,
+                                     repository: repository, arch: arch,
+                                     limit: limit)
+
+    local_jobs_history = []
+    results.elements('jobhist').reverse_each do |result|
+      local_jobs_history << LocalJobHistory.new(rev: result['rev'],
+                                                bcnt: result['bcnt'],
```

Comment:
Does it make sense to make this a more readable key? I have no idea what bcnt is?